template.tpl

```diff
@@ -267,6 +271,18 @@ ___TEMPLATE_PARAMETERS___
             "type": "EQUALS"
           }
         ]
+      },
+      {
+        "type": "LABEL",
+        "name": "showSpace_label",
+        "displayName": "If you would like to open particular Messenger space.",
```

Comment:
I think this is missing an "a" 😄 `If you would like to open a particular Messenger space.`

Reply:
Yep, changed now, thanks!